src/CMakeLists.txt

```diff
@@ -40,6 +40,12 @@ endif()
 add_subdirectory(libraries/fakevim)
 add_subdirectory(libraries/diff_match_patch)
 
+option(USE_QLITE_HTML "Build using system installed Botan 2 crypto library" OFF)

```

Comment:
What has USE_QLITE_HTML to do with Botan?

Reply:
Copy paste ;)

Reply:
I'll change that to "QLiteHtml" then... 😉